core/src/main/java/org/kiwix/kiwixmobile/core/di/modules/ActivityModule.kt

```diff
@@ -35,13 +35,15 @@ import org.kiwix.kiwixmobile.core.utils.AlertDialogShower
 import org.kiwix.kiwixmobile.core.utils.DialogShower
 
 @Module
-abstract class ActivityModule {
-  @Binds
+class ActivityModule {
+
+  @Provides
   @ActivityScope
-  abstract fun bindDialogShower(alertDialogShower: AlertDialogShower): DialogShower
+  internal fun providesDialogShower(activity: Activity): DialogShower = AlertDialogShower(activity)
```

Comment:
revert this change, an abstract class with an `@Binds` method didn't need changing